ProgressCustomCommand now uses Java Time api

dmatej opened the pull request:
- The former syntax was confusing and overcomplicated
- the test detachOnesAttachMulti did not check job result, now does
- Stabilized tests

dmatej committed:
ProgressCustomCommand now uses Java Time api instead of magic

- The former syntax was confusing and overcomplicated
- the test detachOnesAttachMulti did not check job result, now does

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>

dmatej committed:
Further improvements

- measuring the usual latency before time based test
- sleep before attaches
- added check for the progress - for some reason attachers are very slow

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>

OndroMih commented on appserver/tests/admin/admin-extension/src/main/java/com/sun/enterprise/tests/progress/ProgressCustomCommand.java:
```diff
@@ -17,174 +17,101 @@
+    /**
+     * Value must be in for {@code [Nx][MINSEC-]MAXSEC}
+     */
+    @Param(primary = true, separator = ',')
```
Before it was optional and now it's mandatory? Do I understand it correctly?

OndroMih commented on appserver/tests/admin/admin-extension/src/main/java/com/sun/enterprise/tests/progress/ProgressCustomCommand.java:
```diff
@@ -17,174 +17,101 @@
-        ps.setTotalStepCount(getStepCount());
+
+        final ProgressStatus progressStatus = context.getProgressStatus();
+        final List<Duration> parsedIntervals = Stream.of(intervals).map(Duration::parse).toList();
```
What happens if Duration.parse cannot parse input?

dmatej committed:
Faster asadmin in tests, optimized ProgressCustomCommand based tests

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>

OndroMih commented on appserver/tests/admin/admin-extension/src/main/java/com/sun/enterprise/tests/progress/ProgressCustomCommand.java:
```diff
@@ -17,174 +17,101 @@
+                    Thread.currentThread().interrupt();
+                }
+                String message = "Finished block [" + blockId + "] " + intervalText;
+                LOG.log(INFO, message);
```
OK, I didn't realize it's just for tests :)

dmatej committed:
Fixed typo in Xss value and removed verbose gc logging

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>

dmatej committed:
Use same max count of threads as the server thread pool

- It doesn't make much sense to use more

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>

dmatej committed:
Fixed HttpLoadGenerator logic

- requestCounter was increased even when nothing else happened in the iteration
- countRunning changed to Semaphore

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>
